src/components/views/elements/PowerSelector.js

```diff
@@ -74,7 +74,7 @@ module.exports = React.createClass({
         // This needs to be done now because levelRoleMap has translated strings
         const levelRoleMap = Roles.levelRoleMap(newProps.usersDefault);
         const options = Object.keys(levelRoleMap).filter((l) => {
-            return l === undefined || l <= newProps.maxValue;
+            return l === undefined || l <= newProps.maxValue || l == newProps.value;
```

Comment:
while you're here, it might be a good opportunity to not call the variable `l`. I spent probably 15 minutes this morning trying to figure out how 1 can be undefined.

Reply:
Ah yeah, sorry! I have renamed it.